common/predictive-text/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "keyman-lmlayer-prototype",
-  "version": "0.0.0",
+  "name": "keyman-lmlayer",
```

Comment:
Haha, it's been a prototype all this time, and it's been released in two major versions! 😂

Reply:
It occurs to me that we should probably be going with `@keymanapp/lexical-model-layer`. to match `@keymanapp/lexical-model-types` and `@keymanapp/lexical-model-compiler`. There are some additional namespacing concerns, that arise as we use npm more, such as ... perhaps we should be `@keymanapp/developer-lexical-model-compiler` instead of `@keymanapp/lexical-model-compiler` and `@keymanapp/common-lexical-model-types` etc so that the dependency map doesn't become too tangled in our own minds!

Reply:
I think I'll hold off on those latter points for now.

At what point do we not namespace things?  I just checked [Sentry's setup](https://github.com/getsentry/sentry-javascript#supported-platforms) as a reference, and it seems like all of their 'current' JS repos are all 'namespaced' like you suggest.

So, should I do this with `@keymanapp/keyman`, `@keymanapp/keyboard-processor`, etc?

Reply:
I'll go with `@keymanapp/language-modeling-layer` - there's the chance we might eventually incorporate other language-modeling aspects.  That, and I think "language modeling" was the origin of the 'lm' abbreviation anyway.

Reply:
> At what point do we not namespace things?

Uh ... as I said in the next line: we have `lexical-model-compiler` not `developer-lexical-model-compiler`. Given the lexical model compiler is a part of Developer, it makes sense to namespace it accordingly. But no rush on that.

> I'll go with `@keymanapp/language-modeling-layer`

Why not `lexical-model-layer` so it matches `lexical-model-compiler`? What you've proposed just adds more terms and confusion and breaks the mental link between the components. We should be naming according to current use (esp. since you noted 'chance', 'might', 'eventually' which sounds pretty doubtful!)